store/sqlstore/post_store.go

```diff
@@ -1129,11 +1129,10 @@ func (s *SqlPostStore) GetPostsByIds(postIds []string) store.StoreChannel {
 	})
 }
 
-func (s *SqlPostStore) GetPostsBatchForIndexing(startTime int64, endTime int64, limit int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var posts []*model.PostForIndexing
-		_, err1 := s.GetSearchReplica().Select(&posts,
-			`SELECT
+func (s *SqlPostStore) GetPostsBatchForIndexing(startTime int64, endTime int64, limit int) ([]*model.PostForIndexing, *model.AppError) {
+	var posts []*model.PostForIndexing
+	_, err := s.GetSearchReplica().Select(&posts,
+		`SELECT
 				PostsQuery.*, Channels.TeamId, ParentPosts.CreateAt ParentCreateAt
```

Comment:
Nit: this should have the proper indentation like before the PR. One tab less for each line of the string except for the select line.

Reply:
Thank you for the feedback @jespino, I have updated my branch to fix the indentation. Well, I hope I have anyway